### ISSUE TITLE
Lexicon - Avoid ARIA misuse

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1592]
**Describe the bug**
The secondary Settings nav menu is a <div> which uses the ARIA attribute aria-labelledby to point to the H2 heading of the secondary nav menu for its accessible name. However, The aria-labelledby attribute is not allowed on the generic role such as a <div> element.

**To Reproduce**
1. Go to Pulsar Lexicon
2. Click on secondary nav menu such as Lexicon
3. Inspect the container div to identify the incompatible aria-labelledby attribute

**Expected behavior**
When a page implements ARIA, the [ARIA specification](https://www.w3.org/TR/wai-aria-1.2/) should be followed to avoid any issues or confusion for users relating to the misuse of ARIA. Valid ARIA attributes are used for appropriate roles so assistive technologies can interpret and operate the element accurately for users.

**Screenshots**

![Image](https://github.com/user-attachments/assets/9663f878-94b9-48bc-9663-da89666b935f)

**Additional context**
Related success criteria:
[4.1.2 Name, Role, Value (A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Suggested fixes**
Move aria-labelledby from the div container element to the secondary nav menu ul element
